edx_proctoring/templates/timed_exam/submitted.html

```diff
@@ -10,5 +10,10 @@
     {% blocktrans %}
       Your grade for this timed exam will be immediately available on the <a href="{{progress_page_url}}">Progress</a> page.
     {% endblocktrans %}
+    {% if has_due_date %}
+      {% blocktrans %}
+        Once the exam due date has passed, you will be able to once again see your submitted exam answers but not change them.
```

Comment:
Thanks, but "time has run out" is a bit ambiguous because there are two time gates here: the allowed minutes a user has to complete the exam as well as a 'due date'. This message is specific to the due date having to pass before the end-user can see the exam.

Would this be an OK compromise?

"After the due date has passed, you can review the exam, but you cannot change your answers."